src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -141,7 +141,9 @@ def setup_cdk_params(self, config: dict) -> dict:
             "mlNodeCount": self.args.ml_node_count,
             "dataNodeStorage": self.args.data_node_storage,
             "mlNodeStorage": self.args.ml_node_storage,
-            "jvmSysProps": self.args.jvm_sys_props
+            "jvmSysProps": self.args.jvm_sys_props,
+            "use50PercentHeap": str(self.args.use_50_percent_heap).lower(),
+            "isInternal": config["Constants"]["isInternal"]
```

Comment:
Perhaps add a comment indicating what this flag does?

Reply:
Since these go into the cluster-cdk command, the details of all the flags are present in the repo README if someone wants more details.